generators/common/types.d.ts

```diff
@@ -1,17 +1,28 @@
 import type { HandleCommandTypes } from '../../lib/command/types.js';
 import type {
-  Application as BaseApplicationApplication,
   Config as BaseApplicationConfig,
   Entity as BaseApplicationEntity,
-  Field as BaseApplicationField,
   Options as BaseApplicationOptions,
-  Relationship as BaseApplicationRelationship,
   Source as BaseApplicationSource,
 } from '../base-application/types.js';
+import type { Application as DockerApplication } from '../docker/types.js';
+import type {
+  Application as JavascriptApplication,
+  Entity as JavascriptEntity,
+  Field as JavascriptField,
+  Relationship as JavascriptRelationship,
+} from '../javascript/types.js';
+import type { Application as LanguagesApplication, Entity as LanguagesEntity } from '../languages/types.js';
 import type command from './command.ts';
 
 type Command = HandleCommandTypes<typeof command>;
 
+export type Field = JavascriptField & {
```

Comment:
Javascript is common?

Reply:
Yes for now, every client-server application have a package.json with some scripts. Even skipClient.
May be a partial type in the future.